tsdb/db.go

```diff
@@ -762,7 +762,7 @@ func (db *DB) Compact() (err error) {
 			break
 		}
 		mint := db.head.MinTime()
-		maxt := rangeForTimestamp(mint, db.head.chunkRange)
+		maxt := rangeForTimestamp(mint, db.head.chunkRange.Load())
```

Comment:
`chunkRange` only wrote on the init stage. Is it need to refactor?

Reply:
I think it's behind the lock, that's why it was not atomically changed/loaded so far. I think Load is fine for this though